vault/changelog.early-seed.md

```diff
@@ -2,29 +2,40 @@
 id: 3abd00eb-1c1e-4253-aaf5-dcbe20c21850
 title: Early Seed
 desc: ''
-updated: 1653064872154
+updated: 1653672794318
 created: 1604539200840
 published: true
 nav_exclude: true
 ---
 
 > This page documents the features, enhancements, and fixes in the latest early seed release. The early seed is a preview of the weekly release, so you'll see the next minor version when installing (ex. `0.78.0` instead of `0.77.1`). When dendron ships the general release, the currently installed extension will automatically be updated (but will still have same version).
 
-## 0.96.0
+## 0.97.0
 
-Dendron 0.96 has sprouted 🌱
+Dendron 0.97 has sprouted 🌱
 
-**Graph Panel:** We're experimenting with a new note graph panel that lets you visualize relationships between notes. Check it out in the Side Bar!
 
-**Self Contained Vaults:** Self contained vaults is the new way Dendron is handling vaults. Starting with this release, new vaults will now default to being self contained. 
+**Task Note Status:** Wait, did I finish that task? Dendron now displays the status of a linked task note in preview and on published pages so you are always up-to-date with your tasks. 
+
+**Handlebar Templates:** Dendron adds a powerful tool for templating: [Handlebars](https://handlebarsjs.com/guide/), enabling you to use variables, conditionals, for loops and more in your notes. 
+
+**Publishing Configuration**: This release also extends your ability to customize the Side Nav of your published notes. You can now configure whether or not a published note should show it's children in the tree view, allowing you to prevent overcrowding or undesired navigational destinations. 
 
-- More information on [[self contained vaults|dendron://dendron.dendron-site/dendron.topic.vaults.self-contained]]
 
 ### Features
```

Comment:
i've moved around a bunch of the features/enhancements/fixes and changed the wording. can you sync and use the latest text?